server/huntflow_reloaded/tokens.py

```diff
@@ -0,0 +1,99 @@
+""" JWT module """
+import os
+import sys
+from datetime import datetime, timedelta
+from uuid import uuid4
+
+import jwt
+
+if sys.argv[0].split('/')[-1] == 'testing.py':
```

Comment:
Move it to `testing.py`. The current module mustn't know anything about `testing.py`.

Reply:
This line helps the module to understand, that it is calling from tests - the `sys.argv[0]` output is `.../virtualenv/lib/python3.6/site-packages/tornado/testing.py`. If specify this environment variables in module with test cases they are not used.

Reply:
So, write a comment like the following
```
Define the environment variables when the tornado/testing.py is being called.
```